Source/CheckoutAPIClient.swift

```diff
@@ -182,8 +182,7 @@ public class CheckoutAPIClient {
         completion: @escaping ((Swift.Result<CkoCardTokenResponse, NetworkError>) -> Void)
     ) {
         let urlStr = "\(environment.urlPaymentApi)\(Endpoint.tokens.rawValue)"
-        let jsonEncoder = JSONEncoder()
```

Comment:
FYI : Old stale encoder was causing the issue. this one did not use convertFromSnakeCase causing the address line 1 and 2 to never post.